src/libraries/System.Runtime/tests/System.IO.FileSystem.Tests/Enumeration/MatchCasingTests.cs

```diff
@@ -10,6 +10,12 @@ public abstract class MatchCasingTests : FileSystemTest
     {
         protected abstract string[] GetPaths(string directory, string pattern, EnumerationOptions options);
 
+        [Fact]
+        public void EnumerationOptionsNull()
```

Comment:
Where are the tests validating the path and searchPattern arguments result in exception when null?

Reply:
Well, the linked issue and fix code was about `Directory`, and the test is testing `DirectoryInfo`. Test coverage should be added for both paths.

Reply:
Actually this test is covering both `Directory` and `DirectoryInfo`. The coverage should be fine now.